docker/multi-process/scripts/init

```diff
@@ -65,6 +72,16 @@ case "${DATABASE_ADAPTER}" in
   *) echo "Unsupported database adapter. Available adapters are mysql2, and postgresql." && exit 1 ;;
 esac
 
+# Fixup the Procfile and prepare the PORT
```

Comment:
Is there a reason you moved it from the previous location?

Reply:
The IP and PORT to bind are determined by environment variables passed in when creating the container, and will not change if supervisor restarts foreman. So I think this is a more logical place to make the decision once, instead of let foreman.sh to potentially do this every time it somehow needs to restart. There is probably no difference in reality.

I actually don't quite see the need to dynamically generate `foreman.conf`, `foreman.sh`, `bootstrap.conf`, and `bootstrap.sh` via so many HERE docs in the `init` script -- why not just install them as separate scripts?

Reply:
I agree that makes sense, do you want to work on it in a new pull request after we are done with this one?

Reply:
Yes, that shouldn't be too tricky.